### PR TITLE
Remove Flow in favor of pure XML HTTP Request

### PR DESCRIPTION
**Pull request by willrnch**
This removes flow for file transfer to plaza. Instead, we simply make POST request with the file content in the request body. This reduces the number of HTTP requests and simplify the plaza API.

### CODE REVIEW

File: webapp/app/components/vdi-drag-n-drop/component.js [hunk 1]
```diff
@@ -1,22 +1,78 @@
...
+  startUpload() {
+    let req = new XMLHttpRequest();
+
+    this.set('req', req);
+
+    req.open('POST', '/upload?filename=' + encodeURIComponent(this.get('file').name));
+    req.setRequestHeader('Authorization', 'Bearer ' + this.get('token'));
+
+    req.upload.onprogress = (event) => {
+      console.log(...arguments);
```
Comment (dynamiccast) [inline webapp/app/components/vdi-drag-n-drop/component.js]: Remaining console log

**Comment by Gentux** [comment]
You should rebase your work on top of _master_ before making your pull request

File: webapp/app/components/single-tab/component.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ export default Ember.Component.extend({
...
   },
 
+  dragEnter() {
+    console.log('drag enter');
+  },
+
+  dragOver() {
+    console.log('drag over');
+  },
+
```
Comment (spacewoox) [inline webapp/app/components/single-tab/component.js]: console log

**Comment by Gentux** [comment]
Several issues in this PR:
- We can't open file dialog when clicking on link in _files_ tab
- No refresh after uploads is complete, we must refresh the page manually
- Uploading several file result in "blue screen", see attached image
  capture du 2016-06-01 19-32-38

**Pull request merged**